drivers/spi/spi_context.h

```diff
@@ -322,8 +391,10 @@ void spi_context_buffers_setup(struct spi_context *ctx,
 }
 
 /*
- * Note: dfs is the number of bytes needed to store a data frame,
- * while len is the number of data frames sent.
+ * Should be called to update the tracking of TX being completed.
+ *
+ * @param dfs is the number of bytes needed to store a data frame.
```

Comment:
You will need to be consistent: either you document like we do for other apis, or you don't. But you can't mix. 

Actually it might be a good thing to do it as for other APIs. Just my opinion here, if you don't want to it's ok this is quite good stuff already all this documentation.

Reply:
since this is not doxygen, probably not necessary to use doxygen style. I put @param here because the meaning of the parameters of this function were one of the most confusing points for me at first, but I think I'll remove it to be consistent then